RealmSwift/Tests/ObjectCreationTests.swift

```diff
@@ -261,7 +261,7 @@ class ObjectCreationTests: TestCase {
         XCTAssertEqual(stringObjects.count, 1)
         let persistedObject = object.object!
 
-        XCTAssertEqual(persistedObject.intCol, 12)
+        XCTAssertEqual(persistedObject.intCol, 11)
```

Comment:
Oh, I see what it's doing now. The tests would be more understandable if testing for what happens with conflicting values was done in a separate test that tests only that.